charts/clearblade-iot-enterprise/charts/cb-haproxy/templates/haproxy-statefulset.yaml

```diff
@@ -146,8 +146,10 @@ spec:
             limits:
               cpu: {{ .Values.limitCPU }}
               memory: {{ .Values.limitMemory }}
+          {{- if not .Values.global.gdcEnabled }}
```

Comment:
since this is the only options that gdcEnabled controls, maybe we call this variable something else more directly related?